skimage/measure/tests/test_regionprops.py

```diff
@@ -71,6 +71,22 @@ def test_ndim():
         regionprops(np.zeros((10, 10, 10, 2), dtype=np.int))
 
 
+def test_feret_diameter():
+    # comparator result is based on SAMPLE from manually-inspected computations
+    comparator_result = 17.029386365926403
+    test_result = regionprops(SAMPLE)[0].feret_diameter
+    assert_almost_equal(comparator_result, test_result)
+    # square, test that Feret diameter is sqrt(2) * square side
+    img = np.zeros((20, 20), dtype=np.uint8)
+    img[2:-2, 2:-2] = 1
+    feret_diameter = regionprops(img)[0].feret_diameter
+    assert np.abs(feret_diameter - 16 * np.sqrt(2)) < 1
+    # 3d example
+    img_3d = np.dstack((img,) * 3)
```

Comment:
@emmanuelle can you make this a separate test?

Reply:
@emmanuelle The test here looks like it'll stack up and leave the 1 region without boundaries on the third axis. Could you instead do the following?

```
img_3d = np.zeros((20, 20, 3), dtype=np.uint8)
img_3d[2:-2, 2:-2, 1] = 1
```

Reply:
Sorry, strike this comment...I've just realized that the 3D case won't be subject to the same preparatory transformations as the 2D case! 😳